cmd/clone.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/user"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/tentwentyfive/envosaurus/specs"
+	"gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/plumbing/transport/ssh"
+)
+
+var repoConfig string
+
+// cloneCmd represents the clone command
+var cloneCmd = &cobra.Command{
+	Use:   "clone",
+	Short: "Clone repositories",
+	Long:  `Clone repositories`,
+	Run: func(cmd *cobra.Command, args []string) {
+		var projects specs.ProjectsSpec
+		if err := projects.LoadProjects(repoConfig); err != nil {
+			fmt.Println("Error loading projects")
```

Comment:
Checkout the `log` package, I think what you want here is `log.Fatalln()`

https://golang.org/pkg/log/#Fatalln

Reply:
Actually I take that back.  This is a command line application, the log package is used for logging and would include timestamps in the output.  I doubt we want that here.

:ignore_me: